src/views/PerPrioritizationForm/index.tsx

```diff
@@ -416,15 +417,27 @@ export function Component() {
             </Container>
             {actionDivRef.current && (
                 <Portal container={actionDivRef?.current}>
-                    <Button
-                        name={undefined}
-                        variant="secondary"
-                        onClick={handleFormSubmit}
-                        disabled={savePerPrioritizationPending
-                            || readOnlyMode}
-                    >
-                        {strings.saveLabel}
-                    </Button>
+                    {value.is_draft === false ? (
```

Comment:
What is value.is_draft is undefined? Shouldn't it be treated as false?

Reply:
No, is_draft will be undefined when creating the prioritization